### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by adel-gu**
# In this project we did: 

- Chose the Meals DB external API.
- We build the home page and the popup window.
- Builds the home page header and footer.
- Shows the list of items and the number of items on the screen.
- Shows the popup window when the comment button is clicked.
- Used the Involvement API to store and show the item likes, when the user clicks the like button.
- Used the Involvement API to store and show the item comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
+```shell
+npm run test
+```
 
 ### Deployment
+```shell
+npm run deploy
+```
 
 ## Author
```
Comment (rloterh) [inline README.md:47]: - [ ] OPTIONAL
I think you should change the header title _Author_ to _Authors_, as this project is worked on by two contributors.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,48 @@
...
+- JavaScript
+- Webpack
 
-## Live Demo (if available)
-[Experience a live Demo 🚀](https://livedemo.com)
+## Live Demo 
+[Experience a live Demo 🚀](https://adel-gu.github.io/Meals-API/)
 
+## A brief description on the project
+[Video description](https://drive.google.com/file/d/1cW2jRlLERSZEJq7JpNkH4vNYuovwiHtB/view?usp=sharing)
```
Comment (rloterh) [inline README.md:25]: - [ ] Good job with the project presentation, however, you missed some steps in your video presentation. Your video presentation only showed the functional aspect of your app and failed to highlight any portion of your code. Kindly ensure that the video contains a highlight about a piece of the code that you find interesting or that you were proud of making.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Vitalis & Adel, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Correct Github flow used
- ES6 syntax correctly used
- API fetch properly implemented using promise feature
- _Comment_ and _like_ counters properly implemented
- Reviewed project milestones in turns
- Good test coverage using Jest framework
- ReadMe looks professional

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,49 @@
...
-document.body.appendChild(component());
+const handleCommentClick = async (meals) => {
+  const [id, name, category, image, origin] = meals;
+  await populateComment(id, name, category, image, origin, urls);
+  const thepopup = document.getElementById('popup');
+  document.body.classList.add('body-y-scroll');
+  thepopup.classList.add('popup2');
+};
+
+// event.preventDefault();
```
[the remark this answers is not in the document]
Comment (adel-gu) [reply src/index.js:48]: Hi @rloterh, thank you for mentioning that.

File: README.md (hunk 1 above)
Comment (adel-gu) [reply README.md:47]: Definitely we will fix that, thank you  👏

**Review by levy002 — APPROVED** [review]
### Hello @adel-gu  & @svitalis123 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure next time the video presentation must not go over 5 minutes maximum. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**